omod/src/main/webapp/resources/scripts/conditionlist/controllers/manageconditions.controller.js

```diff
@@ -84,5 +102,6 @@ function ManageConditionsController($scope, RestfulService, CommonFunctions) {
     $scope.deactivateCondition = self.deactivateCondition;
     $scope.formatDate = CommonFunctions.formatDate;
     $scope.getConditions = self.getConditions;
-    $scope.conditionConfirmation = self.conditionConfirmation
-}
\ No newline at end of file
+    $scope.redirectToEditCondition = self.redirectToEditCondition;
```

Comment:
What are we using the above line for?

Reply:
To redirect to edit conditions page when user clicks the pencil icon